android/capacitor/src/main/java/com/getcapacitor/plugin/Camera.java

```diff
@@ -335,6 +335,8 @@ private void returnResult(PluginCall call, Bitmap bitmap, Uri u) {
       returnBase64(call, exif, bitmapOutputStream);
     } else if (settings.getResultType() == CameraResultType.URI) {
       returnFileURI(call, exif, bitmap, u, bitmapOutputStream);
+    } else if (settings.getResultType() == CameraResultType.BASE64NOMETADATA) {
```

Comment:
Looks like you made changes on `CameraResultType` class, but didn't commit them. And you also have to do the changes on the types in @capacitor/core